package.json

```diff
@@ -172,7 +172,6 @@
     "got": "^11.8.5",
     "graphql": "^16.6.0",
     "handlebars": "^4.7.3",
-    "https-pem": "^3.0.0",
```

Comment:
Reviewer note:
This devDep was used to provide a self-signed TLS cert to use from some HTTPS testing. It was problematic because it generated the cert in an npm postinstall script, which is icky, and it used 1024 bits for an RSA key, which is too small for FIPS 140.

The test suite had 3-4 separate places where static certs or cert generation was being used for tests. I've consolidated all those into one place: test/fixtures/certs/
It uses a 2048-bit RSA key, which satisfies the strict FIPS OpenSSL provider.